addons/mail/static/src/messaging/component/follower/follower.scss

```diff
@@ -20,6 +19,8 @@
     align-items: center;
     display: flex;
     flex: 1;
+    padding-inline-start: map-get($spacers, 3);
+    padding-inline-end: map-get($spacers, 2);
```

Comment:
It looks better when symmetrical, I would use the same value on both (probably 2?).

Also I noticed that the name link does not have rounded corners, but the two other buttons do, this is a bit strange.

Reply:
Ok for symetry but I don't see any rounded corners

Reply:
As I said on the other PR as the issue is the same here, you can zoom in to see it if you don't notice immediately. I'm sure there are border radius that shouldn't be there.

Reply:
Looking into it on friday, should be due to community too I think.